templates/resources/vmaas_load_balancer_virtual_server.md.tmpl

```diff
@@ -0,0 +1,28 @@
+---
+layout: ""
+page_title: "hpegl_vmaas_load_balancer_virtual_server Resource - vmaas-terraform-resources"
+subcategory: {{ $arr := split .Name "_" }}"{{ index $arr 1 }}"
+description: |-
+  {{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# Resource hpegl_vmaas_load_balancer_virtual_server
+
+-> Compatible version >= 5.4.6
+
+{{ .Description | trimspace }}
+`hpegl_vmaas_load_balancer_virtual_server` resource supports NSX-T Load Balancer Virtual Server creation.
+
+->  Load Balancer ID Data Source `hpegl_vmaas_load_balancer` is supported for Creating Virtual Server.
```

Comment:
this should ideally come in your field description. U cannot add a note for saying there is a data Source. Please refer existing documents.

https://registry.terraform.io/providers/HPE/hpegl/latest/docs/resources/vmaas_network

Reply:
removed